ui/app/styles/components/sidebar.scss

```diff
@@ -102,3 +102,32 @@
     }
   }
 }
+// remove above once MenuSidebar component has been removed
+// new sidebar styling
+.brand-icon {
+  .hs-icon {
+    width: 44px;
+    height: 44px;
+  }
+}
+
+.hc-app-frame {
+  font-family: inherit;
+}
+
+.hc-nav-subnav-item-link.hds-button {
+  font-family: inherit;
+}
+
+.hc-nav-container {
+  .popup-menu-content {
+    .menu-label {
+      font-size: 14px;
```

Comment:
it pains me a little to see hard coded and not variable font sizing. However, I understand our current sizing does not give you 14px. Bulma does use is-2, is-3, etc to define size for fonts. You'll see this in our title.is-5 class naming. just a comment, nothing I think you should change.

Reply:
Ya I think maybe we're in a bit of limbo between moving to HDS and perhaps 14 is a size in that system? Not sure but I took the size from the designs and I think they wanted it to be slightly larger than the link font size which is 13. Hopefully when we move on from Bulma and onto HDS we can address these things with design and have common sizes for everything.